nixos/modules/programs/sway.nix

```diff
@@ -87,20 +89,35 @@ in {
       type = with types; listOf package;
       default = with pkgs; [
         swaylock swayidle alacritty dmenu
-        rxvt-unicode # For backward compatibility (old default terminal)
       ];
       defaultText = literalExample ''
-        with pkgs; [ swaylock swayidle rxvt-unicode alacritty dmenu ];
+        with pkgs; [ swaylock swayidle alacritty dmenu ];
       '';
       example = literalExample ''
         with pkgs; [
-          xwayland
-          i3status i3status-rust
-          termite rofi light
+          # To run Qt5 applications natively on Wayland:
+          qt5.qtwayland
+          # Status bars:
+          i3status i3status-rust waybar
+          # Notification daemon:
+          mako
+          # Screenshots
+          grim slurp sway-contrib.grimshot
+          # dmenu replacements:
+          bemenu rofi
+          # Redshift
+          gammastep wlsunset
+          # Alternative terminals:
+          foot termite
+          # Backlight management:
+          brightnessctl light
```

Comment:
I guess it's highly debatable what to include here :o We could also leave this empty and maybe install `qt5.qtwayland` by default?

Reply:
It's just an example I guess, so don't think this is controversial in any way.